components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -691,8 +728,9 @@ - (void)strokeRotationCycleFinishedFromState:(MDCActivityIndicatorState)state {
     return;
   }
   if (state == MDCActivityIndicatorStateIndeterminate) {
-    if (_cycleColors.count > 0) {
-      _currentColorCount = (_currentColorCount + 1) % _cycleColors.count;
+    NSUInteger cycleColorsCount = self.cycleColors.count;
+    if (cycleColorsCount > 0) {
+      _currentColorCount = (_currentColorCount + 1) % cycleColorsCount;
       [self updateStrokeColor];
```

Comment:
Assuming _cycleColors is never empty:
_currentColorCount = (currentColorCount + 1) % self.cycleColors.count;
Consider s/_currentColorCount/_currentColorIndex/